wayfarer-nomination-status-history.user.js

```diff
@@ -362,7 +362,7 @@
                     if (nom.status != saved.status) {
                         history.push({ timestamp: Date.now(), status: nom.status });
                         // For most status updates, it's also desired to send a notification to the user.
-                        if (nom.status !== 'HELD' && saved.status !== 'HELD') {
+                        if (nom.status !== 'HELD' && (saved.status !== 'HELD' && nom.status !== 'NOMINATED')) {
```

Comment:
I occasionally hit an issue where I don't get a notification when something goes into voting. This typically occurs when I:
- Hold something and make sure the status change has been recorded
- Release the hold from another device, not recording the status change
- The nomination changes from a NOMINATED state to something else

As-is, since the saved status is still HELD, I get no notification, even though the nom status has moved to a notification-worthy status. This ensures that if the status was HELD and becomes NOMINATED (hold released) the nomination still does not trigger a notification, but if the status was HELD, becomes NOMINATED, then becomes something else, a notification does occur.